packages/calcite-components/src/components/list/list.tsx

```diff
@@ -160,6 +160,16 @@ export class List
    */
   @Prop({ reflect: true }) loading = false;
 
+  /**
+   * Specifies the fields to match against when filtering. If not set, all fields will be matched.
```

Comment:
Can we add a note about the default fields being used for filtering (`label`, `description`, `metadata`, `value`)? cc @geospatialem

Reply:
Yep. will do